common/utils/src/test/scala/org/apache/spark/util/StructuredLoggingSuite.scala

```diff
@@ -57,12 +58,14 @@ trait LoggingSuiteBase
 
   def msgWithMDCAndException: LogEntry = log"Error in executor ${MDC(EXECUTOR_ID, "1")}."
 
-  def expectedPatternForBasicMsg(level: Level): String
-
   def msgWithConcat: LogEntry = log"Min Size: ${MDC(MIN_SIZE, "2")}, " +
     log"Max Size: ${MDC(MAX_SIZE, "4")}. " +
     log"Please double check."
 
+  def expectedPatternForBasicMsg(level: Level): String
+
+  def expectedPatternForBasicMsgWithException(level: Level): String
```

Comment:
Add a new `test` for `basicMsg` + `Exception`

Reply:
Nit: Let's add comments about each pattern is about

Reply:
Okay